crates/circuit/src/operations.rs

```diff
@@ -311,13 +309,13 @@ static STANDARD_GATE_NAME: [&str; STANDARD_GATE_SIZE] = [
     "cu",           // 39
     "cu1",          // 40
     "cu3",          // 41
-    "c3x",          // 42
+    "mcx",          // 42 ("mcx")
     "c3sx",         // 43
-    "c4x",          // 44
+    "c4x",          // 44 ("mcx")
     "dcx",          // 45
     "ccz",          // 46
     "rccx",         // 47
-    "rc3x",         // 48
+    "rcccx",        // 48 ("rc3x")
```

Comment:
Same here shouldn't this match the python side name and be `rc3x`?

Reply:
One would think so (that's how I originally called it), but the python side name is `rcccx` :sweat_smile: https://github.com/Qiskit/qiskit/blob/419f40e9e72b4981a6536d08d6b0fa91c13021d5/qiskit/circuit/library/standard_gates/x.py#L874